functests/test_suite_test.go

```diff
@@ -124,6 +125,7 @@ var _ = AfterSuite(func() {
 		namespaces.DpdkTest,
 		sctp.TestNamespace,
 		sriovNamespaces.Test,
+		xt_u32.TestNamespace,
```

Comment:
This should be namespaces.XT_U32Test (also, the name without _ as per the linter suggestion)